CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### 1.0.19
+#### Query strings
+
+- Handle properly query strings on the library
```

Comment:
provide a more detailed list of what this new release will contain.

Also you haven't bumped the version in the package.json file